test/Microsoft.TemplateEngine.TestTemplates/test_templates/ConfigurationKitchenSink/.template.config/template.json

```diff
@@ -2,6 +2,7 @@
   "author": "Test Asset",
   "classifications": [ "Test Asset" ],
   "name": "ConfigurationKitchenSink",
+  "tags": { "type": "project" },
```

Comment:
just wondering whether we should add `$schema` property, since we cleaning this up.